### PR TITLE
fix: pybind11 class definition ordering

### PR DESCRIPTION
**Pull request by vishwa2710**

### CODE REVIEW

File: bindings/python/test/types_/test_integer.py [hunk 1]
```diff
@@ -43,7 +43,6 @@ def test_hash():
 def test_integer_operators():
     a = Integer(15)
     b = Integer(30)
-    c = Integer(-45)
```
Comment (kyle-cochran) [inline bindings/python/test/types_/test_integer.py:46]: Is this on purpose?

File: bindings/python/test/filesystem/test_file.py [hunk 2]
```diff
@@ -41,7 +41,6 @@ def test_file_exists():
 def test_file_getter_functions():
     current_file = File.path(Path.parse("./__init__.py"))
     shared_lib_file = File.path(Path.parse("./shared_library.so.3.3"))
-    test_file = File.path(Path.current() + Path.parse("../assets/pythontestfile.txt"))
```
Comment (kyle-cochran) [inline bindings/python/test/filesystem/test_file.py:44]: Is this on purpose?

File: bindings/python/src/OpenSpaceToolkitCorePy/Type/Real.cpp [hunk 3]
```diff
@@ -2,15 +2,15 @@
 
 #include <OpenSpaceToolkit/Core/Type/Real.hpp>
 
-inline void OpenSpaceToolkitCorePy_Type_Real(pybind11::module& aModule)
+using ostk::core::type::Integer;
+using ostk::core::type::Real;
+using ostk::core::type::String;
+
+inline void OpenSpaceToolkitCorePy_Type_Real(pybind11::class_<Real>& realClass)
```
Comment (kyle-cochran) [inline bindings/python/src/OpenSpaceToolkitCorePy/Type/Real.cpp:9]: Not fully following what's going on here, but will this work for modules that have sub-modules?

In some files like Phyics [Environment](https://github.com/open-space-collective/open-space-toolkit-physics/blob/main/bindings/python/src/OpenSpaceToolkitPhysicsPy/Environment.cpp#L198)

We need a reference to the module in order to instantiate sub-modules. Is it possible to still do this?

**Review by kyle-cochran — COMMENTED** [review]
Couple small questions

File: bindings/python/test/filesystem/test_file.py (hunk 2 above)
Comment (vishwa2710) [reply bindings/python/test/filesystem/test_file.py:44]: Yeah it was unused variable.

**Review by kyle-cochran — APPROVED** [review]

**Pull request merged**